src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -54,13 +56,20 @@ export class HttpConnection implements IConnection {
     private readonly logger: ILogger;
     private readonly options: IHttpConnectionOptions;
     private transport?: ITransport;
+    private transferFormat?: TransferFormat;
     private startPromise?: Promise<void>;
     private stopError?: Error;
     private accessTokenFactory?: () => string | Promise<string>;
+    private connectionId: string | null;
```

Comment:
```suggestion
    private connectionId?: string;
```

Reply:
I had it that way, but then the connectionID is really `string | undefined`. Do we prefer that over `string | null`?